### PR TITLE
Feature/oe 183 increase max w throughout system

### PR DESCRIPTION
**Pull request by Ron-Junior**
## Goal

Resize all pages to occupy all size of the screen

## Review

For the submitter, initial self-review:

- [ ] Reviewed the test cases added for completeness and possible points for discussion
- [ ] Check the scope of the changeset - is everything in the diff required for the pull request?

For the pull request reviewer(s), this changeset has been reviewed for:

- [ ] Consistency between the changeset and the goal stated above
- [ ] Usage friction - is the proposed change in usage cumbersome or complicated?
- [ ] Complexity - is the proposed change introducing any unjustified complexity? 
- [ ] Thoroughness of added tests and any missing edge cases

### CODE REVIEW

**Review by iammayron — CHANGES_REQUESTED** [review]

File: resources/views/castle/departments/create.blade.php [hunk 1]
```diff
@@ -1,18 +1,17 @@
...
     <div>
-        <div class="max-w-6xl mx-auto py-5 sm:px-6 lg:px-8">
+        <div class="max-w-8xl mx-auto py-5 sm:px-6 lg:px-8">
             <a href="{{ route('castle.departments.index') }}" class="inline-flex items-center pt-1 border-b-2 border-green-base text-sm font-medium leading-5 text-gray-900 focus:outline-none focus:border-green-base transition duration-150 ease-in-out">
                 < New Department
             </a>
         </div>
-        <div class="max-w-4xl mx-auto sm:px-6 lg:px-8">
+        <div class="max-w-8xl mx-auto sm:px-6 lg:px-8">
             <x-form :route="route('castle.departments.store')" post>
```
Comment (iammayron) [inline resources/views/castle/departments/create.blade.php:9]: You don't need this `post` attribute, this is already the default type.

**Review by iammayron — CHANGES_REQUESTED** [review]

**Review by lucas-devsq — CHANGES_REQUESTED** [review]
Add 8xl for width in tailwind configurations

**Review by lucas-devsq — APPROVED** [review]
